src/components/app/content/index.js

```diff
@@ -53,15 +58,25 @@ function Content() {
         setCards(cards);
     }
 
+    function addCard(cardData) {
+        const generatedId = cards.map((item) => item.id).sort((a, b) => b - a)[0] + 1
```

Comment:
Weird logic. The `uuid` package has been proposed to you in the task assignment, Better use it.